pages/general-information-and-resources/glossary.md

```diff
@@ -82,7 +83,7 @@ redirect_from:
 
 - **BOC** - Budget Object Code. Used for accounting.
 
-- **BPR** - Business Process Reengineering. This is when an organization designs
+- **BPR** - Business Process Re-engineering. This is when an organization designs
```

Comment:
I'll look into how to configure cSpell later, but I'm making this as a note for myself: We'll stick with "Reengineering" because that's how the Government Accountability Office (GAO) styles it.